The CI have many errors. I checked 2 of them and it was:
```
In file included from tensorflow/core/common_runtime/gpu/gpu_cudamallocasync_allocator.cc:22:0:
./tensorflow/core/common_runtime/gpu/gpu_cudamallocasync_allocator.h:21:10: fatal error: third_party/gpus/cuda/include/cuda.h: No such file or directory
 #include "third_party/gpus/cuda/include/cuda.h"
```

I just pushed a fix that should fix those Google CI build error. The build was working here.

--- a/tensorflow/core/common_runtime/gpu/gpu_cudamallocasync_allocator.cc
+++ b/tensorflow/core/common_runtime/gpu/gpu_cudamallocasync_allocator.cc
@@ -33,18 +33,20 @@ namespace tensorflow {
 GpuCudaMallocAsyncAllocator::GpuCudaMallocAsyncAllocator(
     PlatformGpuId platform_gpu_id, size_t pool_size,
     bool reserve_memory, bool compute_stats)
-    : cuda_stream_(nullptr),
-      name_(absl::StrCat("gpu_async_", platform_gpu_id.value())) {
+    : name_(absl::StrCat("gpu_async_", platform_gpu_id.value())) {
   stream_exec_ =
       DeviceIdUtil::ExecutorForPlatformDeviceId(GPUMachineManager(),
                                                 platform_gpu_id).ValueOrDie();
 
 #if CUDA_VERSION < 11020
   LOG(FATAL) << "TF_GPU_ALLOCATOR=cuda_malloc_async need CUDA 11.2 or higher to compile.";
+#elif !defined(GOOGLE_CUDA)
+  LOG(FATAL) << "GOOGLE_CUDA not defined";
 #else
   // Initialized here as it only exist if compiled with a recent
   // enough CUDA.
   pool_ = nullptr;
+  cuda_stream_ = nullptr;
   // WAR an CUDA 11.2 driver bug for multiple-GPU. It currently
   // request that the context on GPU 0 is initialized. Which isn't the
   // case for TF+horovod.
@@ -115,7 +117,9 @@ GpuCudaMallocAsyncAllocator::GpuCudaMallocAsyncAllocator(
 }
 
 GpuCudaMallocAsyncAllocator::~GpuCudaMallocAsyncAllocator() {
+#ifdef GOOGLE_CUDA
   cuStreamDestroy(cuda_stream_);
+#endif
 }
 
 void* GpuCudaMallocAsyncAllocator::AllocateRaw(size_t alignment,

--- a/tensorflow/core/common_runtime/gpu/gpu_cudamallocasync_allocator.h
+++ b/tensorflow/core/common_runtime/gpu/gpu_cudamallocasync_allocator.h
@@ -18,8 +18,10 @@ limitations under the License.
 
 #include <memory>
 
+#ifdef GOOGLE_CUDA
 #include "third_party/gpus/cuda/include/cuda.h"
 #include "third_party/gpus/cuda/include/cuda_runtime_api.h"
+#endif  // GOOGLE_CUDA
 
 #include "absl/container/flat_hash_map.h"
 #include "tensorflow/core/common_runtime/gpu/gpu_id.h"
@@ -74,14 +76,16 @@ class GpuCudaMallocAsyncAllocator : public Allocator {
   // compute stream and already synchronize with the h2d, d2h and d2d
   // stream. So we do not need to ask cudaMallocAsync to add extra
   // synchronization.
+#if defined(GOOGLE_CUDA) && CUDA_VERSION >= 11020
   cudaStream_t cuda_stream_;
-  string name_;
+
   //Not owned. The default pool of the associated GPU.
   //If null, then the instanciation failed and the first allocation
   //will return an error.
-#if CUDA_VERSION >= 11020
   CUmemoryPool pool_;
-#endif
+#endif  // GOOGLE_CUDA
+
+  string name_;
 
   TF_DISALLOW_COPY_AND_ASSIGN(GpuCudaMallocAsyncAllocator);
 

